tests/test-values.yaml

```diff
@@ -0,0 +1,31 @@
+isWorkloadCluster: false
+isManagementCluster: false
```

Comment:
For now we test a configuration that is neither `WorkloadCluster` nor `ManagementCluster` because the simple `smoke` test scenario does not include other components that would be needed on a cluster to simulate the environment. However, let's look into `functional` tests after this.

Reply:
Good point, you can try to install an ingress controller with an app fixture ([Example code](https://github.com/giantswarm/cert-exporter/blob/master/tests/ats/test_cert_exporter.py#L113-L124)

Reply:
Thank you for the example, that's really useful!